smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftSymbolProvider.kt

```diff
@@ -380,14 +380,14 @@ class SwiftSymbolProvider(private val model: Model, val swiftSettings: SwiftSett
             if (shape.hasTrait<StreamingTrait>()) {
                 { writer ->
                     writer.format(
-                        "\$N.data(\$N(\"$literal\".utf8))",
+                        "\$N.data(\$N(base64Encoded: \"$literal\"))",
```

Comment:
Smithy blob default values are always base64-encoded strings so the default is decoded here.